commons/src/main/java/org/infinispan/commons/hash/MurmurHash3.java

```diff
@@ -432,28 +432,28 @@ private long MurmurHash3_x64_64_String(String s, long seed) {
          }
 
          if (cp <= 0x7f) {
-            addByte(state, (byte) cp, ++byteLen);
+            addByte(state, (byte) cp, byteLen++);
```

Comment:
This looks like changing the result - how could the test pass (produce equal results) with the first commit only?